data/Other/postinst

```diff
@@ -190,10 +190,10 @@ Height=0
     	configtemplate=${configtemplate/POSITION/"center_right"}
 
     elif [[ $halign =~ "end" ]] && [[ $valign =~ "end" ]]; then
-    	configtemplate=${configtemplate/POSITION/"bottom_right"}    	
+    	configtemplate=${configtemplate/POSITION/"bottom_right"}
     fi
 
 
     echo "$configtemplate" >$wallpaper"config"
 
-done	
\ No newline at end of file
+done
```

Comment:
This whole script is something that should be rethought and probably removed, but that can come in a future step.

Reply:
Agreed.